src/pages/Home.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Home = () => (
+  <div className="home">
+    <h1 className="welcome">Welcome to our page!</h1>
+    <p className="para">
+      Mathematics, the science of structure, order, and relation that has
+      evolved from elemental practices of counting, measuring, and describing
+      the shapes of objects. It deals with logical reasoning and quantitative
+      calculation, and its development has involved an increasing degree of
+      idealization and abstraction of its subject matter. Since the 17th
+      century, mathematics has been an indispensable adjunct to the physical
+      sciences and technology, and in more recent times it has assumed a similar
+      role in the quantitative aspects of the life sciences.
+    </p>
+    <p className="para">
+      In many cultures—under the stimulus of the needs of practical pursuits,
+      such as commerce and agriculture—mathematics has developed far beyond
+      basic counting. This growth has been greatest in societies complex enough
+      to sustain these activities and to provide leisure for contemplation and
+      the opportunity to build on the achievements of earlier mathematicians.
+    </p>
```

Comment:
- [ ] Optional suggestion

- You can modify the font-size of the text to look nice as well as for us UI/UX purpose.

Reply:
I appreciate. 
Thank you.